tests/test_uri/test_validator/test_urivalidator.py

```diff
@@ -523,13 +523,13 @@ def test_is_remote_when_authority_is_null(self):
     #     self.assertEqual(authority.number_case(), UAuthority.NumberCase.ID)
 
     # def test_is_remote_when_authority_has_name_and_number_set(self):
-    #     authority = UAuthority(name="vcu.veh.gm.com", id=b"hello Jello")
+    #     authority = UAuthority(name="vcu.veh.com", id=b"hello Jello")
```

Comment:
We don't have to remove gm.com references, only the other word :-)

Reply:
donedone